src/client/App.js

```diff
@@ -42,6 +43,10 @@ function App() {
         <Route exact path="/contact-us">
           <ContactUsPage />
         </Route>
+        <Route exact path="/favorites">
+          <FavoritePage />
+        </Route>
+
```

Comment:
FavoritePage should be in AuthentificatedRoute.

Reply:
will do when authentication is done